src/sidebar/components/Annotation/Annotation.js

```diff
@@ -57,7 +58,11 @@ function Annotation({
   const showActions = !isSaving && !isEditing;
   const showReplyToggle = !isReply && !hasAppliedFilter && replyCount > 0;
 
-  const onReply = () => annotationsService.reply(annotation, userid);
+  const onReply = () => {
+    if (annotation && isSaved(annotation)) {
+      annotationsService.reply(annotation, userid);
+    }
+  };
```

Comment:
This callback is only used if the annotation is not being edited and not being saved, so the added check here is strictly redundant, but it's much easier to see now that `reply` will never be passed an unsaved annotation.

Reply:
I appreciate this as it expresses an expectation that was implicit before.